gettsim/transfers/wohngeld.py

```diff
@@ -432,9 +440,10 @@ def wohngeld_miete_bis_2008(
 
     data = [
         wohngeld_params["max_miete"][hh_größe][constr_year][ms]
-        if hh_größe <= 5
+        if hh_größe <= (list(wohngeld_params["max_miete"].keys())[4])
```

Comment:
```suggestion
        if hh_größe <= max(wohngeld_params["max_miete"])
```
Should also work like that. Since the variable is needed several times, it would also make sense to define `max_definierte_hh_größe = max(wohngeld_params["max_miete"])` and make use of this variable.

Reply:
It seems not to work like that, I only get error messages...